docs/i18n.rst

```diff
@@ -0,0 +1,181 @@
+   "POT-Creation-Date: 2017-10-06 19:21+0200\n"
+   "PO-Revision-Date: 2017-10-11 15:02+0200\n"
+   "Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+   "Language: de\n"
+   "Language-Team: de <LL@li.org>\n"
+   "Plural-Forms: nplurals=2; plural=(n != 1)\n"
+   "MIME-Version: 1.0\n"
+   "Content-Type: text/plain; charset=utf-8\n"
+   "Content-Transfer-Encoding: 8bit\n"
+   "Generated-By: Babel 2.3.4\n"
+
+The first thing you should do is to fill out the ``Last-Translator`` field with
+your contact information. You may also want to do the same with the copyright
+notice at the top of the document.
+
+The remainder of the file is were translations are actually defined:
+
+.. code-block:: none
+
+   #: botogram/defaults.py:46
+   msgid "Use /help to get a list of all the commands."
+   msgstr ""
+
+Each message is assigned a ``msgid`` string which identifies it across
+translations: in botogram it is the English translation for that message.
+``msgstr`` fields are instead specific to each translation and define that
+message's translation for the file's language: this is where you need to enter
+your translation. If a ``msgstr`` is empty (as they are by default) botogram
+will default to the English translation.
+
+Some messages could contain HTML formatting or
+`Python string interpolation`_: your translation should reflect these as closely
+as possible. If you need context on the usage of a message, you can refer to its
+usages in the source code included in the comment line above each string.
+
+Just to be sure your syntax is correct, you can ensure your translation will
+compile correctly by invoking ::
+
+   $ invoke i18n-compile
+
+If the command succeeds there's good chance you didn't mess up anything.
+
+Once you're done, you can commit and push your changes to your fork and propose
+them to be merged into the upstream repository to be included in the next
+botogram release.
+
+.. _i18n-update-translation:
+
+Updating a translation
+======================
+
+As botogram evolves, more message will probably be added to the codebase, and it
+is also possible for currently included translations to contain mistakes.
+
+The workflow for updating a translation is basically the same as the one
+described in `i18n-new-language`_, but you may also need to use ::
+
+   $ invoke i18n-extract
+
+to extract new messages from the codebase. The command also ensures references
+in comments are up-to-date with their current location.
+
+Running the command will always result in the ``.pot`` file and ``.po`` files
+being updated, at least for what concerns the ``POT-Creation-Date`` header. You
+should check your diff and avoid committing any change that doesn't impact the
+actual translation and the source code references in comments.
+
+While trivial, it would be nice if you also changed the ``PO-Revision-Date``
+header to reflect your changes.
+
+.. _i18n-overriding:
+
+Overriding default messages
+===========================
```

Comment:
Maybe this section should be before the translating ones, I feel it's more important.